projects/packages/search/src/dashboard/class-initial-state.php

```diff
@@ -80,6 +80,8 @@ public function get_initial_state() {
 				'search'                 => $this->module_control->is_active(),
 				'instant_search_enabled' => $this->module_control->is_instant_search_enabled(),
 			),
+			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			'features'        => filter_var_array( explode( ',', $_GET['features'] ), FILTER_SANITIZE_STRING ),
```

Comment:
I wonder if we should go for something a bit more "WordPress-y", that would completely strip out bad strings? something like

```suggestion
			'features'        => array_map( 'sanitize_text_field', explode( ',', $_GET['features'] ) ),
```

What do you think?

Reply:
Sounds good 👍